googleMapsApiModule.py

```diff
@@ -3,14 +3,18 @@
 import config
 import mysql.connector
 
+gmaps = googlemaps.Client(config.key)
+#this line is repeated twice in the code and needs to be globalised
+
+url={'direction':"https://www.google.com/maps/dir/?api=1&" , 'geocoding':"https://www.google.com/maps/search/?api=1&query="}
+#this is a global dictionary 'url' having all the web address required throughtout the code for better understanding
+
 def direction(origin,destination):
-    gmaps = googlemaps.Client(config.key)
     result = gmaps.directions(origin,destination)
     address = "origin="+origin+"&"+"destination="+destination
     address = address.lower()
     address = address.replace(" ","+")
-    url = "https://www.google.com/maps/dir/?api=1&"
-    result_url = url+address
+    result_url = url['direction']+address
```

Comment:
Something like this would be more readable imho:

```python3
result_url = "{}/{}".format(BASE_URL['direction'], address.lower().replace(" ", "+"))
```

Reply:
Good suggestion @athiyakumar. A minor suggestion building on top of what you've suggested, f-strings are the way forward for python string formatting. With f-strings, it would look like this `result_url = f'{BASE_URL['direction']}/{address.lower().replace(" ", "+")}'

Reply:
Great suggestion @janakrajchadha, indeed let's follow with the best practices. @athityakumar, thanks for pointing it out.

Reply:
Great line of code, I will definitely change this.

Reply:
Since this is a beginner level issue, I thought it's best to strictly stick to the scope, I shall be creating a new issue to address all similar string generations (there are multiple instances in the same file). Does that sound good @janakrajchadha and @athityakumar?

Reply:
Only trying to be fair as far as the scoring guidelines are considered, want to be liberal in the points participants earn for the work they put in. :)